app/models/ocelot/Process.scala

```diff
@@ -28,6 +28,8 @@ case class Process(meta: Meta, flow: Map[String, Stanza], phrases: Vector[Phrase
     titleIndex => phraseOption(titleIndex).getOrElse(Phrase(meta.title, meta.title))
   }
   lazy val startUrl: Option[String] = flow.get(Process.StartStanzaId).collect{case ps: PageStanza => ps.url}
+
```

Comment:
lazy val as we want it to executed only once, probably only needed on Meta rather than both

Reply:
I've removed this method from Process.